system/Language/en/Cast.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Cast language strings.
+ *
+ * @package      CodeIgniter
+ * @author       CodeIgniter Dev Team
+ * @copyright    2014-2018 British Columbia Institute of Technology (https://bcit.ca/)
+ * @license      https://opensource.org/licenses/MIT	MIT License
+ * @link         https://codeigniter.com
+ * @since        Version 3.0.0
+ * @filesource
+ * 
+ * @codeCoverageIgnore
+ */
+
+return [
+	'Cast.jsonErrorDepth' 		=> 'Maximum stack depth exceeded',
```

Comment:
Language strings should not have the filename as part of the key here. So - shouldn't be `Cast.jsonErrorDepth`, should be simply `jsonErrorDepth`.

Reply:
done. thx.